### PR TITLE
Unpickle

### PR DESCRIPTION
**Pull request by Atomfusion1**
We Got Pickle to save file.. 
this gives unpickle to load the file back
one line, No trial error to find out why its not loading when it points to EQ instead of eq/config 

2 changes
1. minor changed pickle from void to return bool, return true if load, else false
2. lua_unpickle . getconfig path, merge path if fullname,  convert to double \\ . run lua loadfile

### CODE REVIEW

File: src/plugins/lua/bindings/lua_MQBindings.cpp [hunk 1]
```diff
@@ -529,21 +529,42 @@ static void lua_pickle(sol::this_state L, std::string_view file_path, sol::table
...
+	for (size_t pos = 0; (pos = pathString.find("\\", pos)) != std::string::npos; pos += 2) 
+	{
+		pathString.replace(pos, 1, "\\\\");
+	}
+
+	// Concatenate strings and run Lua code:
+	std::string luaCode =
+		"local fn, err = loadfile('" + pathString + "')\n"
+		"if not fn then error(err or '') end\n"
+		"return fn()";
```
Comment (brainiac) [inline src/plugins/lua/bindings/lua_MQBindings.cpp:564]: If you went this far you might as well just call this on the c++ side. All you're doing here is writing a script that does nothing but call one function, and we can do that better from c++.

**Review by brainiac — CHANGES_REQUESTED** [review]
see comment

**Comment by dannuic** [comment]
At this point, we should probably have a whole pickle namespace to provide all the pickle related functions, dump and load

**Comment by brainiac** [comment]
i don't know if thats necessary...

**Pull request closed without merging**